.github/workflows/cypress.yml

```diff
@@ -0,0 +1,64 @@
+name: '[QE] Cypress Runner'
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 8:30 AM UTC = 4:30 AM EDT
+    - cron: '30 8 * * *'
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths-ignore:
+      - CHANGELOG.md
+      - docs/**
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+    # To avoid waste Cypress tests,
```

Comment:
Please, add a `timeout-minutes: 30` to define the max time this job can consume.